### ISSUE TITLE
paging a string

### ISSUE THREAD

**Issue by kj7rrv** [jonathanslenders/pypager#14]
I have a big multiline string. Can I page it without a generator?

**Comment by jonathanslenders** [comment]
Yes, there is a StringSource: https://github.com/prompt-toolkit/pypager/blob/master/pypager/source.py#L380

Use this instead of GeneratorSource, which is used in the example from the README: https://github.com/prompt-toolkit/pypager

**Comment by kj7rrv** [comment]
Thanks!
